common/containers/Tabs/Swap/components/CurrencySwap.jsx

```diff
@@ -14,34 +15,34 @@ import type {
 import bityConfig, { kindMin, kindMax } from 'config/bity';
 import { toFixedIfLarger } from 'utils/formatters';
 
-export type StateProps = {
-  bityRates: {},
-  originAmount: ?number,
+type Props = {
+  bityRates: any,
+  originAmount: number,
   destinationAmount: ?number,
   originKind: string,
   destinationKind: string,
-  destinationKindOptions: String[],
-  originKindOptions: String[]
-};
-
-export type ActionProps = {
+  destinationKindOptions: string[],
+  originKindOptions: string[],
   originKindSwap: (value: string) => OriginKindSwapAction,
   destinationKindSwap: (value: string) => DestinationKindSwapAction,
-  originAmountSwap: (value: ?number) => OriginAmountSwapAction,
+  originAmountSwap: (value: number) => OriginAmountSwapAction,
   destinationAmountSwap: (value: ?number) => DestinationAmountSwapAction,
-  changeStepSwap: () => ChangeStepSwapAction,
+  changeStepSwap: (value: number) => ChangeStepSwapAction,
```

Comment:
This has value as a required number, but it's optional above. Can it be required everywhere? Not sure we'd ever want to call it with a non-number.